src/test/java/org/candlepin/model/PoolCuratorTest.java

```diff
@@ -1988,7 +1981,7 @@ public void testUpdateQuantityColumnsOnPool() {
 
         Pool pool = createPool(owner, product, 20L,
             TestUtil.createDate(2010, 3, 2), TestUtil.createDate(
-            Calendar.getInstance().get(Calendar.YEAR) + 1, 3, 2));
+                Calendar.getInstance().get(Calendar.YEAR) + 1, 3, 2));
```

Comment:
This one is correct. Each line continuation should be indented. This rule differs from checkstyle, because checkstyle couldn't handle more then one line break for a single row of code.

Reply:
Why would we want an additional intent to process the continuation of an arbitrary parameter that itself is only being line-broken due to the same processing rule leading to the first line break?

The fact that we're in the middle of a nested call is entirely incidental, and doesn't make this blob of code any more readable. It's no more or less informative or arbitrary than the `createDate` on the untouched line that *isn't* broken up, or the subsequent call to `.getInstance().get(...)` and the math that follows. It's just another indentation because a line break happened to occur where it did.

Fun thought: would it still require an indent if the line were to change to read `TestUtil\n.createDate(`?

Reply:
As a follow-on here, a better approach is to break this up into individual named parameters, or build the pool directly instead of using a long multi-parameter call that tends to run into this formatting issue.